donate/templates/fragments/donate_form.html

```diff
@@ -150,7 +110,6 @@ <h2 class="donate-form__actions-heading heading heading--secondary">{% trans "Ch
         {% render_form_field braintree_form.braintree_nonce %}
         {% render_form_field braintree_form.amount %}
         {% render_form_field braintree_form.frequency %}
-        {% render_form_field braintree_form.currency %}
```

Comment:
This is a hidden input which is required by the Paypal JS - see https://github.com/mozilla/donate-wagtail/blob/master/source/js/payments-paypal.js#L8 . Without it the Paypal modal will error when you try to complete payment. Was this deleted by mistake?

Reply:
Yep, looks like a mistake (no idea how that happened!) thanks for spotting it, have re-added.